04_how_to_improve_viz.Rmd

```diff
@@ -0,0 +1,28 @@
+# High quality and inclusive data visualizations
+
+```{r, include = FALSE}
+ottrpal::set_knitr_image_path()
+```
+
+## Learning Objectives
+
+This chapter will cover:
+
+- {LO1}
+- {LO2}
+
+## Visual Design Principles
```

Comment:
Can be helpful to talk about preattentive attributes and how they're used in viz design: https://laderast.github.io/data_storytelling_bdc/#22